flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/utils/TableTestBase.scala

```diff
@@ -1824,6 +1826,11 @@ object TableTestUtil {
     s.replaceAll("\"id\"\\s*:\\s*\\d+", "\"id\" : ").trim
   }
 
+  /** JobJson {jid} is ignored. */
+  def replaceJobId(s: String): String = {
+    s.replaceAll("\"jid\"\\s*:\\s*\"[\\w.-]*\"", "\"jid\":\"\"").trim
```

Comment:
This might also replace a string literal having `"jid..."` pattern. We should make sure that only field values are targeted in this case